packages/katello/katello/katello-remove

```diff
@@ -62,7 +62,7 @@ RPMS=[
   '^pulp',
   '^python-pulp',
   '^pulp-',
-  'mongo',
+  'rh-mongodb',
```

Comment:
Since we grep in `rpm -qa` this is not needed. `mongo` will also match `rh-mongodb` and properly clean up both pre-migration and post-migration systems.